Resources/Prototypes/Body/Mechanisms/basic_human_organs.yml

```diff
@@ -1,5 +1,17 @@
+- type: entity
+  id: BaseOrgan
```

Comment:
Rename this to BaseOrganHuman. Bonus points if you want to separate it into BaseOrganHuman : BaseMechanism : BaseItem but I don't think that's necessary yet.

Reply:
I did just that